src/dotdirdeps.cpp

```diff
@@ -155,6 +155,7 @@ static void drawDirectory(TextStream &t, const DirDef *const directory, const Do
       "fillcolor=\"" << getDirectoryBackgroundColor(directory->level()-startLevel)   << "\", "
       "color=\""     << getDirectoryBorderColor(property)                            << "\", "
       "URL=\""       << addHtmlExtensionIfMissing(directory->getOutputFileBase())    << "\""
+      "tooltip=\""   << directory->briefDescriptionAsTooltip() << "\""
       "];\n";
```

Comment:
I assume this change has to do with #9467 and it looks a bit out of order for this proposed pull request.
I think it would be better to create a separate pull request for this change.

Reply:
Sure. But please review the rest of PR. I try to merge it for two years!